community/kernel/src/main/java/org/neo4j/helpers/collection/Visitable.java

```diff
@@ -19,7 +19,7 @@
  */
 package org.neo4j.helpers.collection;
 
-public interface Visitable<T>
+public interface Visitable<V>
 {
-    void accept(Visitor<T, RuntimeException> visitor);
+    void accept( V visitor );
```

Comment:
Yes, I agree with @davidegrohmann, I did not mean that you should change this interface, I meant that you should complement it, move your `Visitable` interface to possibly the same package, but under a different name, or perhaps to the `org.neo4j.function` package.